apps/snitch_api/lib/snitch_api/address_context.ex

```diff
@@ -0,0 +1,87 @@
+defmodule SnitchApi.AddressContext do
+  @moduledoc """
+  The AddressContext context.
+  """
+
+  import Ecto.Query, only: [from: 2, order_by: 2]
+
+  alias Snitch.Repo
+  alias SnitchApi.API
+  alias Snitch.Data.Schema.Address
+
+  @doc """
+  Returns the list of addresses.
+
+  ## Examples
+
+      iex> list_addresses()
+      [%Address{}, ...]
+
+  """
+  def list_addresses(conn, _params) do
+    Repo.all(Address)
+  end
+
+  @doc """
+  Gets a single address.
+
+  Raises if the Address does not exist.
+
+  ## Examples
+
+      iex> get_address!(123)
+      %Address{}
+
+  """
+  def get_address!(id), do: Repo.get(Address, id)
+
+  @doc """
+  Creates a address.
+
+  ## Examples
+
+      iex> create_address(%{field: value})
+      {:ok, %Address{}}
+
+      iex> create_address(%{field: bad_value})
+      {:error, ...}
+
+  """
+  def create_address(attrs \\ %{}) do
+    Address.changeset(%Address{}, attrs)
+    |> Repo.insert()
```

Comment:
This gives the credo warning for piping.
Either of the following approaches will resolve them.
```elixir
Address.changeset(%Address{}, attrs) |> Repo.insert()
```
```elixir
Address
|> Address.changeset(attrs) 
|> Repo.insert()
```